modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -163,11 +163,9 @@ namespace {{packageName}}.Client
             var response = RestClient.Execute(request);
             {{/supportsUWP}}
             {{#supportsUWP}}
-            // TODO: This is obviously not a valid response, but it prevents compiling-errors
-            // 'RestClient.CallApiAsync(..)' is used instead, because 'RestClient.ExecuteAsync(...)' 
-            // is available for Windows UWP, while 'RestClient.CallApi(...)' is not
-            var response = new object(); 
-            {{/supportsUWP}}
+            // Using async method to perform sync call (uwp-only)
+            var response = RestClient.ExecuteTaskAsync(request).Result;
+			{{/supportsUWP}}
```

Comment:
@jerry1621  Can we use 4-space instead of tab to make it consistent with the rest of of the indention?